### ISSUE TITLE
Mulelog for 18 character accounts?

### ISSUE THREAD

**Issue by mrandros** [kolton/d2bot-with-kolbot#666]
Title pretty much say itself

Is it possible to make mulelog support logging for accounts with 18 characters on it?

At the moment it only log 8

**Comment by vibeD2** [comment]
no, it wont scroll passed the 8 characters you would have to manually scroll down for it.
